docs/en_US/NAS/HardwareAwareNAS.rst

```diff
@@ -23,19 +23,25 @@ Then run multi-trail SPOS demo:
 How the demo works
 ------------------
 
-To support latency-aware NAS, you first need a `Strategy` that supports filtering the models by latency. We provide such a filter named `LatencyFilter` in NNI and initialize a `Random` strategy with the filter:
+To support hardware-aware NAS, you first need a `Strategy` that supports filtering the models by latency. We provide such a filter named `LatencyFilter` in NNI and initialize a `Random` strategy with the filter:
 
 .. code-block:: python
 
-  simple_strategy = strategy.Random(model_filter=LatencyFilter(100)
+  simple_strategy = strategy.Random(model_filter=LatencyFilter(threshold=100, predictor=base_predictor))
 
 ``LatencyFilter`` will predict the models\' latency by using nn-Meter and filter out the models whose latency are larger than the threshold (i.e., ``100`` in this example).
 You can also build your own strategies and filters to support more flexible NAS such as sorting the models according to latency.
 
-Then, pass this strategy to ``RetiariiExperiment`` along with some additional arguments: ``parse_shape=True, dummy_input=dummy_input``:
+Then, pass this strategy to ``RetiariiExperiment`` along with additional argument: ``applied_mutators=[]``:
 
 .. code-block:: python
 
-  RetiariiExperiment(base_model, trainer, [], simple_strategy, True, dummy_input)
+  exp = RetiariiExperiment(base_model, trainer, [], simple_strategy)
```

Comment:
You can write `exp = RetiariiExperiment(base_model, trainer, strategy=simple_strategy)` so that you don't have to explain the stuff about `applied_mutators`.